sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -425,8 +425,9 @@ def list_entities(
         # type: (...) -> ItemPaged[TableEntity]
         """Lists entities in a table.
 
-        :keyword int results_per_page: Number of entities per page in return ItemPaged
+        :keyword int results_per_page: Number of entities per page in returned ItemPaged
         :keyword select: Specify desired properties of an entity to return certain entities
+        :keywork entity_hook: Callable for custom deserialization
```

Comment:
We'll need a type for this, something like:
`:keyword Callable[Mapping]` entity_hook:` (I think it's a dict we're passing in, right?)

Reply:
yes it is a dictionary, thank you for the clarification